### PR TITLE
Stock News Sentiment Analysis

### PR DESCRIPTION
**Pull request by Shobhit-Bandhu**
# Pull Request for ML-Crate 💡

## Issue Title: Stock News Sentiment Analysis #351

- **Info about the related issue (Aim of the project)** : 
- **Name:** Shobhit-Bandhu
- **GitHub ID:** Shobhit-Bandhu
- **Idenitfy yourself:** Contributor KWOC '23

Closes: #351

### Describe the add-ons or changes you've made 📃

Preprocessed data, added SVM model.

## Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested? ⚙️

Describe how it has been tested
Describe how have you verified the changes made

## Checklist: ☑️

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Comment by abhisheks008** [comment]
Please complete the whole project and update the blanks in the Jupyter Notebook and `README.md`. Once you are done with this, do let me know.

File: Stock News Sentiment Analysis/Model/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Stock News Sentiment Analysis
```
Comment (abhisheks008) [inline Stock News Sentiment Analysis/Model/README.md:1]: Please follow the `README.md` template and update it accordingly. Here is the template, https://github.com/abhisheks008/ML-Crate/blob/main/.github/readme_template.md

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
Also check out the errors of Tensorflow modelling and try impose it properly.

**Comment by abhisheks008** [comment]
Hi @Shobhit-Bandhu I have seen that CNNs are not properly implemented in your project, you can remove it from the notebook.

**Comment by abhisheks008** [comment]
Hey @Shobhit-Bandhu your implemented models are not even having accuracy score of 80 atleast. Please upgrade the models and try to achieve an accuracy of atleast 80%.

**Comment by Shobhit-Bandhu** [comment]
@abhisheks008 Any suggestions? Should I use BERT models?

**Comment by abhisheks008** [comment]
> @abhisheks008 Any suggestions? Should I use BERT models?

Yeah you can try that.

**Comment by Shobhit-Bandhu** [comment]
@abhisheks008 I can't seem to do better than 54%...

**Comment by abhisheks008** [comment]
Hi @Shobhit-Bandhu sorry for the delay, as I was in vacation. Can you please update the `README.md` file with the newly added model's accuracy score.

**Comment by Shobhit-Bandhu** [comment]
@abhisheks008 Anything else you would like added?

**Review by abhisheks008 — APPROVED** [review]
LGTM! Your PR is approved and ready to be merged.
@Shobhit-Bandhu

**Pull request merged**